README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Thank you Jorge.
After some test later today, I'm going to do another commit based on your review.
1. Modify README.md to describe network mode clearly.
2. Modify robot_auto_blending.cpp by adding failure counter.
3. Move industrial_robot_simulator  from service mode to client mode.
4. Add execute privilege to irb2400_client.sh.